feature/manager.go

```diff
@@ -151,6 +172,26 @@ func newManager(cfg Config, desc setDesc) (*Manager, error) {
 			raw.Unset(lnwire.ShutdownAnySegwitRequired)
 		}
 
+		for _, custom := range cfg.CustomFeatures[set] {
+			if raw.IsSet(custom) {
+				return nil, fmt.Errorf("feature bit: %v (%d) "+
+					"already set", custom, custom)
```

Comment:
> currently I am not really understanding why we print out the feature bit 2 times

The formatting directives are different, this will print the numerical value and `String` for the feature bit.

Reply:
> Is it a design decision to not set a feature which is already set, I would prefer logging a warning but returning an error and therefore exiting I find that a bit strict 

It is a design decision. If a user is trying to set feature bits that are already set, then they've misunderstood the way the API works (or perhaps have a bug in their code). I think restricting to sane add/remove instructions for an advanced feature is reasonable (and the way it's already done in the `UpdateNodeAnnouncement` API.

Reply:
Sounds well thought through thanks

Reply:
I tested it to see whats printed out, its basically 2 times the number because custom features are not mapped :

lnd --protocol.custom-nodeann=7
```
unable to create server: feature bit: 7 (7) already set
```
with the following change it would be something like this:
```
			if raw.IsSet(custom) {
				name := lnwire.Features[custom]
				return nil, fmt.Errorf("feature bit: %v (%d) "+
					"already set", name, custom)
			}
```
```
unable to create server: feature bit: gossip-queries (7) already set
```

Reply:
of maybe something like to make sure we do not crash if the name is not in the mapping?

```
if name, ok := lnwire.Features[custom]; ok {

}
```

Reply:
Ah, this was copied across from `peersrpc` where there was a `String` function on the `lnrpc.FeatureBit`. Removed the duplicate, nice catch.

Reply:
nit: I would also include the set but personal taste:

```
unable to create server: feature bit: 0 already set in context: SetInit
```

Reply:
wdym here, `SetInit` is the name of the set?

Reply:
yes I thought we could include the set in which the bit is already set, current output just says bit XX already set, I would prefer the set too, wdyt ?